ggml/src/ggml-cpu/CMakeLists.txt

```diff
@@ -169,6 +169,30 @@ function(ggml_add_cpu_backend_variant_impl tag_name)
                 # Raspberry Pi 3, 4, Zero 2 (32-bit)
                 list(APPEND ARCH_FLAGS -mno-unaligned-access)
             endif()
+            if (${CMAKE_SYSTEM_PROCESSOR} MATCHES "aarch64")
+                message(STATUS "Linux aarch64 detected")
+
+                set(MARCH_FLAGS "-march=armv8.2-a+fp16")
```

Comment:
This assumes 2 things:
1. Every `aarch64` CPU supports fp16. I think the code should look for `fphp` CPU feature.
2. `aarch64` will be at least `armv8.2`, in fact `aarch64` implies only `armv8`